src/components/SegmentedControl/SegmentedControl.test.tsx

```diff
@@ -136,7 +136,13 @@ describe('Segmented Control Component', () => {
       fireEvent.click(screen.getByRole('listitem', { name: clickedOption.key }))
 
       expect(onChange).toBeCalledTimes(1)
-      expect(onChange).toBeCalledWith(clickedOption)
+      expect(onChange).toBeCalledWith(clickedOption, expect.objectContaining({
```

Comment:
Also here

Reply:
Same approach adopted in the other comment